src/utils/getPackages.js

```diff
@@ -5,5 +5,9 @@ export default async function getPackages(opts: { cwd?: string } = {}) {
   let cwd = opts.cwd || process.cwd();
   let project = await Project.init(cwd);
   let workspaces = await project.getWorkspaces();
-  return workspaces.map(workspace => workspace.pkg.dir);
+  return workspaces.map(workspace => ({
```

Comment:
Updated as discussed. 
Cant see any usages of this, and @d4rkr00t  hasn't used it anywhere, so should be a safe change